.github/workflows/version.yml

```
@@ -1,113 +1,113 @@
 name: Create a new version
 
 on:
-    push:
-        branches: [master]
+  push:
+    branches: [master]
 
 jobs:
-    version:
-        runs-on: macos-latest
+  version:
+    runs-on: macos-latest
 
-        steps:
-            # Version: 2.3.4
-            - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
-              with:
-                  fetch-depth: 0
-                  token: ${{ secrets.OS_BOTIFY_TOKEN }}
+    steps:
+      # Version: 2.3.4
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
 
-            - uses: softprops/turnstyle@8db075d65b19bf94e6e8687b504db69938dc3c65
-              with:
-                  poll-interval-seconds: 10
-              env:
-                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: softprops/turnstyle@8db075d65b19bf94e6e8687b504db69938dc3c65
+        with:
+          poll-interval-seconds: 10
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-            - name: Set up git
-              run: |
-                git fetch
-                git checkout master
-                git config user.name OSBotify
+      - name: Set up git
+        run: |
+          git fetch
+          git checkout master
+          git config user.name OSBotify
 
-            - name: Checkout new branch
-              run: git checkout -b version-bump-${{ github.sha }}
+      - name: Checkout new branch
+        run: git checkout -b version-bump-${{ github.sha }}
 
-            - name: Generate version
-              id: bumpVersion
-              uses: Expensify/Expensify.cash/.github/actions/bumpVersion@master
-              with:
-                  GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+      - name: Generate version
+        id: bumpVersion
+        uses: Expensify/Expensify.cash/.github/actions/bumpVersion@master
+        with:
+          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
 
-            - name: Commit and tag new version
-              run: |
-                git add \
-                  ./package.json \
-                  ./package-lock.json \
-                  ./android/app/build.gradle \
-                  ./ios/ExpensifyCash/Info.plist \
-                  ./ios/ExpensifyCashTests/Info.plist
-                git commit -m "Update version to ${{ steps.bumpVersion.outputs.newVersion }}"
-                git tag ${{ steps.bumpVersion.outputs.newVersion }}
+      - name: Commit and tag new version
+        run: |
+          git add \
+            ./package.json \
+            ./package-lock.json \
+            ./android/app/build.gradle \
+            ./ios/ExpensifyCash/Info.plist \
+            ./ios/ExpensifyCashTests/Info.plist
+          git commit -m "Update version to ${{ steps.bumpVersion.outputs.VERSION }}"
```

You forgot to update this to `NEW_VERSION`
```suggestion
          git commit -m "Update version to ${{ steps.bumpVersion.outputs.NEW_VERSION }}"
```